src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+            "preferredDockedSize": {"width": 230, "height": 300},
+            "preferredFloatingSize": {"width": 230, "height": 300},
+            "icons": [
+                {"width":23,"height":23,"path":"icons/dark.png","scale":[1,2],"theme":["darkest","dark","medium"]},
+                {"width":23,"height":23,"path":"icons/light.png","scale":[1,2],"theme":["lightest","light"]}
+            ]
+        }
+    ],
+    "icons": [
+        { "width": 23, "height": 23, "path": "icons/icon_D.png", "scale": [ 1, 2 ], "theme": [ "dark", "darkest" ], "species": [ "generic" ] },
+        { "width": 23, "height": 23, "path": "icons/icon_N.png", "scale": [ 1, 2 ], "theme": [ "lightest", "light" ], "species": [ "generic" ] }
+    ],
+    "featureFlags": {
+        "enableExperimentalWebviewSupport" : true
+    },
+    "requiredPermissions": {
+        "network": {
+            "domains": [
+                "https://adobe.com",         
+            ]
+        },
+        "clipboard": "readAndWrite",
+        "webview": {
+            "allow": "yes",
+            // domains --> string[] | "all"
+            "domains": [ "https://*.adobe.com", "https://*.google.com"],
+            "allowLocalRendering": "yes"
+        },
+        "launchProcess": {           
+            "schemes": 
+                [ "https", "slack"],            
+            "extensions": 
+                [ ".xd", ".ps" ],        
+        }, 
+    },
+}
+```
+
+## Changes to top-level metadata
+Starting from changes to top-level metadata, here are some keys that changed with the v5 upgrade. Read [manifest v4](../manifest-v4/) to learn more about each key/value field.
+
+Key path | Type | Description | Change 
+---------|------|-------------|-------
+`version` | string| Version number of your plugin in `x.y.z` format. | Plugins can specify semver format with no warnings. <br/>Specify at least one number and the minor and/or patch will be autofilled with zeroes.
+`title` | string | Title for the plugin. <br/>Defaults to plugin's name if none is specified | New in v5.
+`requiredPermissions` | object | Declare [plugin permissions](#plugin-permissions). | New in v5.
+`entrypoints` | `EntryPointDefinition[]` | Describes the entries your plugin adds to the _Plugins_ menu and plugin panel. | v5 changes in next section.
+
+## Changes to entry points
+The changes to `entryPoints` add flexibility in specifying the initial view/location of plugin panels.
+
+Key | Type | Description | Change
+----|------|------------ |------
+`id` | string | Unique identifier for the entry point. This `id` will also be mapped to entrypoints defined in your plugin code. | Passed as the id for the uxpcommand, uxpshowview (New V5) and  uxphideview (New V5) events.
+`hostUIContext` | object | Use [this property](#hostuicontext) to customize how entrypoints appear. | New in v5.
+`featureFlags` | object | Indicates feature flags in use. | `enableWebview` boolean flag. 
+
+### HostUIContext 
```

Comment:
Although HostUIContext is added for 3P devs, we do not have the support from Ps/XD yet. Please remove it.